src/components/dashboard/Dashboard.js

```diff
@@ -200,13 +202,16 @@ const Dashboard = props => {
   const subscribeToFeed = async () => {
     await getFeedPage(true)
 
-    userStorage.feedEvents.on('updated', onFeedUpdated)
+    userStorage.feedStorage.feedEvents.on('updated', onFeedUpdated)
   }
 
-  const onFeedUpdated = event => {
-    log.debug('feed cache updated', { event })
-    getFeedPage(true)
-  }
+  const onFeedUpdated = useCallback(
+    debounce(event => {
```

Comment:
Lodash's debounce doesn't working correctly with the hooks. the solution is:

```javascript
const defaultDelay = 500

const useDebouncedCallback = (callback, options = null) => {
   const { delay = defaultDelay, leading = false } = options || {}
   const callbackRef = useRef(null)  
   const timeoutRef = useRef(null)
   const suppressFlagRef = useRef(false)

   useEffect(() => (callbackRef.current = callback), [callback])

   return useCallback(() => {
     const invoke = () => isFunction(callbackRef.current) && callbackRef.current()

     if (timeoutRef.current) {
       clearTimeout(timeoutRef.current)
       timeoutRef.current = null
     }

     if (!leading) {
       timeoutRef.current = setTimeout(invoke, delay)
       return
     }

     if (!suppressFlagRef.current) {
       suppressFlagRef.current = true
       invoke()
     }

     timeoutRef.current = setTimeout(() => (suppressFlagRef.current = false), delay)
   }, [delay, leading])
 }
```

Reply:
we discussed this in the past. it works just fine as long the method is not being re-rendered.